# TYR CATCH
# foobar()
# print('after try')

print('----------------------')
try:
   foobar
except:
   print('POBLEM!')

print('after try')